front/src/Stores/BiggestAvailableAreaStore.ts

```diff
@@ -1,112 +1,115 @@
 import { get, writable } from "svelte/store";
 import type { Box } from "../WebRtc/LayoutManager";
 import { HtmlUtils } from "../WebRtc/HtmlUtils";
-import { LayoutMode } from "../WebRtc/LayoutManager";
-import { embedScreenLayout } from "./EmbedScreensStore";
+import { highlightedEmbedScreen } from "./EmbedScreensStore";
 
 /**
  * Tries to find the biggest available box of remaining space (this is a space where we can center the character)
  */
 function findBiggestAvailableArea(): Box {
     const game = HtmlUtils.querySelectorOrFail<HTMLCanvasElement>("#game canvas");
-    if (get(embedScreenLayout) === LayoutMode.VideoChat) {
-        const children = document.querySelectorAll<HTMLDivElement>("div.chat-mode > div");
-        const htmlChildren = Array.from(children.values());
-
-        // No chat? Let's go full center
-        if (htmlChildren.length === 0) {
-            return {
-                xStart: 0,
-                yStart: 0,
-                xEnd: game.offsetWidth,
-                yEnd: game.offsetHeight,
-            };
-        }
+    const wholeScreenBox = {
+        xStart: 0,
+        yStart: 0,
+        xEnd: game.offsetWidth,
+        yEnd: game.offsetHeight,
+    };
 
-        const lastDiv = htmlChildren[htmlChildren.length - 1];
-        // Compute area between top right of the last div and bottom right of window
-        const area1 =
-            (game.offsetWidth - (lastDiv.offsetLeft + lastDiv.offsetWidth)) * (game.offsetHeight - lastDiv.offsetTop);
-
-        // Compute area between bottom of last div and bottom of the screen on whole width
-        const area2 = game.offsetWidth * (game.offsetHeight - (lastDiv.offsetTop + lastDiv.offsetHeight));
-
-        if (area1 < 0 && area2 < 0) {
-            // If screen is full, let's not attempt something foolish and simply center character in the middle.
-            return {
-                xStart: 0,
-                yStart: 0,
-                xEnd: game.offsetWidth,
-                yEnd: game.offsetHeight,
-            };
-        }
-        if (area1 <= area2) {
-            return {
-                xStart: 0,
-                yStart: lastDiv.offsetTop + lastDiv.offsetHeight,
-                xEnd: game.offsetWidth,
-                yEnd: game.offsetHeight,
-            };
+    const blockers: Box[] = [];
+
+    const screenOnTop = get(highlightedEmbedScreen)?.embed;
+    if (screenOnTop) {
+        //@ts-ignore
```

Comment:
Could you remove the ts-ignore of this PR?

Here, the ".embed" property is defined as a `EmbedScreen` type which is:

http://github.com/thecodingmachine/workadventure/blob/6dd04e340c5f8896b72e2a6edb107b6f28943ea4/front/src/Stores/EmbedScreensStore.ts#L7-L16

```
export type EmbedScreen =
    | {
          type: "streamable";
          embed: Streamable;
      }
    | {
          type: "cowebsite";
          embed: CoWebsite;
      };
```

the iframe property exists only on `CoWebsite`, so you can remove the typescript warning by testing this:

```
if (screenOnTop.type === "cowebsite")
```